Gemfile.lock

```diff
@@ -557,7 +557,7 @@ GEM
       thor (>= 0.18.1, < 2.0)
     rainbow (2.0.0)
     raindrops (0.13.0)
-    rake (10.4.2)
+    rake (10.5.0)
```

Comment:
Thanks @munckymagik I agree with that, tests pass fine but like you say it will be easier to roll back if necessary if its an isolated commit